### PR TITLE
docs(ee/conf) fix stream_listen config

### PR DESCRIPTION
**Pull request by agarwal-akash**
* updated the property description for stream_listen in ee/0.36-x

### CODE REVIEW

**Comment by agarwal-akash** [comment]
Fixes #1527

File: app/enterprise/0.36-x/property-reference.md [hunk 1]
```diff
@@ -269,6 +269,13 @@ proxy_url = https://127.0.0.1:8443
 **Description**
 
 Comma-separated list of addresses and ports on which stream mode should listen.
+This value accepts IPv4, IPv6, and hostnames.
+Some suffixes can be specified for each pair:
+- `proxy_protocol` will enable usage of the
```
Comment (h3xar0n) [inline app/enterprise/0.36-x/property-reference.md]:
```suggestion
- `proxy_protocol` enables usage of the
```

File: app/enterprise/0.36-x/property-reference.md [hunk 2]
```diff
@@ -269,6 +269,13 @@ proxy_url = https://127.0.0.1:8443
...
+  PROXY protocol for a given address/port.
+- `transparent` will cause kong to listen to, and
```
Comment (h3xar0n) [inline app/enterprise/0.36-x/property-reference.md]:
```suggestion
- `transparent` will cause Kong to listen to, and
```

File: app/enterprise/0.36-x/property-reference.md [hunk 3]
```diff
@@ -269,6 +269,13 @@ proxy_url = https://127.0.0.1:8443
...
+  respond from, any and all IP addresses and ports
+  you configure in iptables.
```
Comment (h3xar0n) [inline app/enterprise/0.36-x/property-reference.md]:
```suggestion
  you configure in `iptables`.
```

**Pull request merged**